CMakeLists.txt

```diff
@@ -0,0 +1,96 @@
+# Minimum cmake version required by scikit-build-core: 3.15
+# Minimum cmake version required by `Python_SOABI`: 3.17
+cmake_minimum_required(VERSION 3.17)

```

Comment:
If I remember correctly, the default CMake version on Ubuntu 20.04 is 3.16. Do we need to specify Ubuntu 22.04 LTS in the `Requirements` section in the README?